src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -78,7 +78,7 @@ std::string jsonRequestTreat
     if ((answer = parseDataP->ent.res.check(ciP, EntitiesRequest)) != "OK")
     {
       OrionError error(SccBadRequest, answer);
-      return error.render(ciP, "");
+      return setStatusCodeAndSmartRender(ciP, error);
```

Comment:
setStatusCodeAndSmartRender as a function ...
Could it not be a method for OrionError instead?